asciidoctorj-distribution/src/test/groovy/org/asciidoctor/diagram/WhenDitaaDiagramIsRendered.groovy

```diff
@@ -27,8 +27,7 @@ class WhenDitaaDiagramIsRendered extends Specification {
 
         given:
         String imageFileName = UUID.randomUUID()
-        File imagesOutDir = testFolder.newFolder()
-        File createdImage = new File(imagesOutDir, "${imageFileName}.png")
+        File imagesOutDir = new File(testFolder.root, "images-dir")
```

Comment:
Without looking at the codenarc report I guess the build failed because this could be a string with single quotes instead of double quotes.
Also I could imagine that codenarc prefers `.name` over `.getName()` but I am not sure about that.